### ISSUE TITLE
Provide Web interface on hub.gdgx.io 

### ISSUE THREAD

**Issue by friedger** [gdg-x/zeppelin-grunt#2]
Add a UI to generate an event website

**Comment by ozasadnyy** [comment]
@friedger you are talking about service that will generate setting for a site, aka CMS?

**Comment by friedger** [comment]
@ozasadnyy yes, so that you don't have to run grunt locally. Where the site is hosted needs still to be discussed.

**Comment by ozasadnyy** [comment]
It would be a great step for the project, but we need another developers, because I have no idea how to do this.
